the-collab-lab/tcl-19-smart-shopping-list
# Ju jm style itemslist

## Description
TailwindCSS stylesheet was imported into `index.html` and basic responsive styling was added to the `ItemsList` component.  Also, the header was styled which will be used for both components, the `ItemsList` and `AddItems`. For visibility reasons, the header was styled with bg-green-400 instead of bg-green-300. 


## Related Issue
closes #31 

## Acceptance Criteria
X Install TailwindCSS, so it will be shown in `package.json`
✓ The `ItemsList` Component has a basic styling, created with TailwindCSS.
✓ A "Header" styling is created, which can be used for both components, `ItemsList` and `AddItems`.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

![empty list](https://docs.google.com/uc?export=download&id=1VJo3Ms5zYPJXDMSrWj7a02rpLpVsiXde)
![full list](https://docs.google.com/uc?export=download&id=1MX1xSGBd19CdW_i4zCaj5tLRNUnRndN3)


## Testing Steps / QA Criteria
• From your terminal, pull down this branch with "git pull origin ju-jm-style-itemslist" and check that branch out with "git checkout ju-jm-style-itemslist"
• start the server with "npm start"
• Navigate to the `ItemsList` component to view the style when the shopping list is empty
• Also observe the `ItemsList` component when the shopping list contains items (set different values: soon, kind of soon, not soon) and check out the different colors for the items when they are checked/unchecked.
